apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -101,4 +165,9 @@ if (!('BridgeServiceMixin' in self)) {
       { methods: METHODS, streams: STREAMS, events: EVENTS }
     )
   );
+
+  // Automatically init service if it's run inside worker.
+  if ('importScripts' in self) {
```

Comment:
Don't have better ideas right now. Initially it was like the code below, but it's even worse :)
```
  var globalScope = self.constructor.name;
  if (globalScope === 'SharedWorkerGlobalScope' ||
      globalScope === 'DedicatedWorkerGlobalScope') {
    ConversationService.init();
  }
```
Maybe bridge can provide static method like ```inWorker()``` or something. What are your thoughts?